sources/Makefile

```diff
@@ -160,7 +160,7 @@ local-clean:
 # Local DB
 postgres:
 	@echo "+\n++ Make: Run local db.\n+"
-	@docker compose -f docker-compose.yml up postgres
+	@docker compose -f docker-compose.yml up -d postgres
```

Comment:
Thanks for the change. Many times, I had to stop the the docker from terminal and start the container separately to have the DB running.